### PR TITLE
DOCSP-43827 1.8.1 RN

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Release notes for C2C 1.8.1.

## Staging

[RN](https://deploy-preview-430--docs-cluster-to-cluster-sync.netlify.app/release-notes/1.8/#1.8.1-release)

## JIRA
[DOCSP-43827](https://jira.mongodb.org/browse/DOCSP-43827)

### CODE REVIEW

**Review by jwilson-mdb — APPROVED** [review]
Thanks @kennethdyer LGTM!

File: source/release-notes/1.8.txt [hunk 1]
```diff
@@ -21,16 +21,21 @@ Patch Releases
...
+- Support added for OpenID Connect (OIDC) authentication with
+  the workload (machine) flow. For more information, see
+  :ref:`c2c-rn-atlas-workload-identity`.
 
-For details, see :ref:`c2c-authentication`.
+Issues Fixed:
+
+- Fixed bug introduced in ``mongosync`` 1.8.0, where
+  ``mongosync`` crashes if it finds a replica set with 
+  :rsconf:`settings.replicaSetId` unset.
```
Comment (FGasper) [inline source/release-notes/1.8.txt:38]: Is it worth linking to the SERVER ticket that caused this? Just so users can have context on whether the issue affects them.

Comment (kennethdyer) [reply source/release-notes/1.8.txt:38]: @FGasper, Generally, we don't link to the JIRA tickets.  But, if you think it's important, I can throw it in.

**Review by FGasper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**